skeleton/cli-bundler/aurelia_project/tasks/run.ext

```diff
@@ -48,9 +58,17 @@ function reload() {
   browserSync.reload();
 }
 
-let run = gulp.series(
+const run = gulp.series(
   serve,
   done => { watch(reload); done(); }
 );
 
-export default run;
+const shutdownAppServer = (code) =>{
+  return new Promise(resolve => {
+    bs.exit();
+    process.exit(code || 0);
```

Comment:
I understand because of the browsersync/socket.io issue you mentioned, you want to force termination. But that issue was fixed in BrowserSync/browser-sync#1121 https://github.com/BrowserSync/browser-sync/commit/4b17d60f82b5de6cb7187511d4554a4311edbb88

Are you sure we still need this patch?

Reply:
`bs.exit()`, stops the `browser-sync`, but there is still something weird the `node` does not stop, and I couldn't know why?! 

<img width="717" alt="Screen Shot 2019-08-21 at 21 40 27" src="https://user-images.githubusercontent.com/4405518/63452814-7c1fb980-c45c-11e9-9540-af6266ac36f1.png">

So, I think for now it would be better to have this patch for now, until we find a solution.

Reply:
I see, I will test this.

Reply:
Got it 😄 

It's not browsersync holds back process, it's our default watch mode in run task!
Modify run task to
```js

const shutdownAppServer = (code) =>{
  return new Promise(resolve => {
    bs.exit();
    resolve();
  });
};

// export serve with the watch
export { run as default , serve, shutdownAppServer };
```

Then do
```js
import { serve, shutdownAppServer } from './run';

export default (cb) => {
  if (CLIOptions.hasFlag('start')) {
    runCypress(cb);
    serve();
    return;
  }

  runCypress(cb);
};
```

All work!

Reply:
> It's not browsersync holds back process, it's our default watch mode in run task!
Modify run task to

Yeah, I said browser-sync works fine, after they closed the issue, I couldn't find a way for our wtach mode, thanks for the update, I'll change it.